pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -330,7 +331,10 @@ export default defineComponent({
     }"
   >
     <slot name="label">
-      <label v-if="hasLabel">
+      <label
+        v-if="hasLabel"
+        :for="labelForIdentifier"
```

Comment:
If an id is provided for labeled input, then the `for` identifier will not match the provided id.

Reply:
We can check if `id` has been provided as an attribute on the `mounted()` hook; use that to update the data prop that makes the `for` connection:

```js
  mounted() {
    const { id = null } = this.$attrs;

    if (id) {
      this.inputId = id;
    }
  },
```

Reply:
That's a great idea! So simple... 🤦  Implemented

I also noticed that `labelForIdentifier` wasn't connected to anything 🤷 🤷 🤷  🤦 

Everything corrected and up for review. Thank you very much for your review mate... 🙇